src/integrator/LBInitPopUniform.cpp

```diff
@@ -24,67 +24,63 @@
 #include "LBInitPopUniform.hpp"
 
 namespace espressopp {
-  namespace integrator {
-//    LOG4ESPP_LOGGER(LBInitPopUniform::theLogger, "LBInitPopUniform");
-    LBInitPopUniform::LBInitPopUniform(shared_ptr<System> system,
-                                       shared_ptr< LatticeBoltzmann > latticeboltzmann)
-    : LBInit(system, latticeboltzmann) {
-    }
-
-    void LBInitPopUniform::createDenVel (real _rho0, Real3D _u0) {
-			if (mpiWorld->rank()==0) {
-				printf("Creating an initial configuration with uniform density %f and \nvelocity %f, %f, %f\n",
-							 _rho0, _u0.getItem(0), _u0.getItem(1), _u0.getItem(2));
-				printf("-------------------------------------\n");
-			} else {
-				// do nothing
-			}
-			
-      real invCs2 = 1. / latticeboltzmann->getCs2();
-
-      real invCs4 = invCs2*invCs2;
-      real scalp, value;
-			int _offset = latticeboltzmann->getHaloSkin();
-			Int3D _Ni = latticeboltzmann->getMyNi();
-			int _numVels = latticeboltzmann->getNumVels();	// number of velocities in the model
-			
-      // set initial velocity of the populations from Maxwell's distribution
-#warning: check if taking care of OFFSET AT INITIALISATION () is needed
-			for (int i = 0; i < _Ni.getItem(0); i++) {
-      // test the damping of a sin-like initial velocities:
-        real trace = _u0*_u0*invCs2;
-        for (int j = 0; j < _Ni.getItem(1); j++) {
-          for (int k = 0; k < _Ni.getItem(2); k++) {
-            for (int l = 0; l < _numVels; l++) {
-              scalp = _u0 * latticeboltzmann->getCi(l);
-              value = 0.5 * latticeboltzmann->getEqWeight(l) * _rho0 * (2. + 2. * scalp * invCs2 + scalp * scalp * invCs4 - trace);
-              latticeboltzmann->setLBFluid(Int3D(i,j,k),l,value);
-              latticeboltzmann->setGhostFluid(Int3D(i,j,k),l,0.0);
+   namespace integrator {
+      //    LOG4ESPP_LOGGER(LBInitPopUniform::theLogger, "LBInitPopUniform");
+      LBInitPopUniform::LBInitPopUniform(shared_ptr<System> system,
+                                         shared_ptr< LatticeBoltzmann > latticeboltzmann)
+      : LBInit(system, latticeboltzmann) {
+      }
+      
+      void LBInitPopUniform::createDenVel (real _rho0, Real3D _u0) {
+         if (mpiWorld->rank()==0) {
```

Comment:
Why erasing the whole (c) here? for one line? look at commit #91 for solving it!